packages/themes/src/MenderNext/common.ts

```diff
@@ -11,63 +11,149 @@
+    200: colorTokens?.variables['purple(M)']['200'].values[themeMode] || purple[200],
+    300: colorTokens?.variables['purple(M)']['300'].values[themeMode] || purple[300],
+    400: colorTokens?.variables['purple(M)']['400'].values[themeMode] || purple[400],
+    500: colorTokens?.variables['purple(M)']['500'].values[themeMode] || purple[500],
+    600: colorTokens?.variables['purple(M)']['600'].values[themeMode] || purple[600],
+    700: colorTokens?.variables['purple(M)']['700'].values[themeMode] || purple[700],
+    800: colorTokens?.variables['purple(M)']['800'].values[themeMode] || purple[800],
+    900: colorTokens?.variables['purple(M)']['900'].values[themeMode] || purple[900]
   },
   cyan: {
     ...cyan,
-    50: colorTokens?.variables['cyan (M)']['50'].values['Mode 1'] || cyan[50],
-    100: colorTokens?.variables['cyan (M)']['100'].values['Mode 1'] || cyan[100],
-    200: colorTokens?.variables['cyan (M)']['200'].values['Mode 1'] || cyan[200],
-    300: colorTokens?.variables['cyan (M)']['300'].values['Mode 1'] || cyan[300],
-    400: colorTokens?.variables['cyan (M)']['400'].values['Mode 1'] || cyan[400],
-    500: colorTokens?.variables['cyan (M)']['500'].values['Mode 1'] || cyan[500],
-    600: colorTokens?.variables['cyan (M)']['600'].values['Mode 1'] || cyan[600],
-    700: colorTokens?.variables['cyan (M)']['700'].values['Mode 1'] || cyan[700],
-    800: colorTokens?.variables['cyan (M)']['800'].values['Mode 1'] || cyan[800],
-    900: colorTokens?.variables['cyan (M)']['900'].values['Mode 1'] || cyan[900]
+    50: colorTokens?.variables['cyan (M)']['50'].values[themeMode] || cyan[50],
+    100: colorTokens?.variables['cyan (M)']['100'].values[themeMode] || cyan[100],
+    200: colorTokens?.variables['cyan (M)']['200'].values[themeMode] || cyan[200],
+    300: colorTokens?.variables['cyan (M)']['300'].values[themeMode] || cyan[300],
+    400: colorTokens?.variables['cyan (M)']['400'].values[themeMode] || cyan[400],
+    500: colorTokens?.variables['cyan (M)']['500'].values[themeMode] || cyan[500],
+    600: colorTokens?.variables['cyan (M)']['600'].values[themeMode] || cyan[600],
+    700: colorTokens?.variables['cyan (M)']['700'].values[themeMode] || cyan[700],
+    800: colorTokens?.variables['cyan (M)']['800'].values[themeMode] || cyan[800],
+    900: colorTokens?.variables['cyan (M)']['900'].values[themeMode] || cyan[900]
+  }
+};
+
+const spacing = (multiplier = 1) => spacingTokens?.variables[multiplier].values[themeMode];
+
+export const components: Components = {
+  MuiAccordion: {
+    styleOverrides: {
+      root: {
+        border: 'none',
+        boxShadow: 'none',
+        '&:before': {
+          display: 'none'
+        },
+        padding: `${spacing(1)}px ${spacing(2)}px`,
+        alignItems: 'flex-start',
+        alignSelf: 'stretch',
+        [`&.${accordionClasses.expanded}`]: {
+          margin: 'auto'
+        }
+      }
+    }
+  },
```

Comment:
```suggestion
      root: ({ theme }) => ({
        border: 'none',
        boxShadow: 'none',
        '&:before': {
          display: 'none'
        },
        padding: theme.spacing(1, 2),
        alignItems: 'flex-start',
        alignSelf: 'stretch',
        [`&.${accordionClasses.expanded}`]: {
          margin: 'auto'
        }
      })
    }
  },
```

maybe? I think we'd need to have proper spacing set in theme anyways, as we use `theme.spacing` in our components.